core/types/index.d.ts

```diff
@@ -496,8 +498,11 @@ declare module '@verdaccio/types' {
     tag?: string;
   }
 
-  type AuthAccessCallback = (error: string | null, access: boolean) => void;
-  type AuthCallback = (error: string | null, groups: string[] | false) => void;
+  // FIXME: error should be export type `VerdaccioError = HttpError & { code: number };` instead of AuthError
+  // but this type is on @verdaccio/commons-api and cannot be used here yet (I don't know why)
```

Comment:
Yeah, that's something I regret now. This types and also commons api will be part of the `5.x` so we won't have this issue anymore eventually.